### ISSUE TITLE
Behavior change in version 4.0 of `lcobucci/jwt` causing `private` authentication failure.

### ISSUE THREAD

**Issue by JingBh** [GrahamCampbell/Laravel-GitHub#128]
Starting from version `4.0.0` of [`lcobucci/jwt`](https://github.com/lcobucci/jwt/tree/4.0.0), it is the default behavior to encode timestamps as `float` (which is discussed [here](https://github.com/lcobucci/jwt/discussions/623)). This is causing an authentication failure as GitHub only accepts `int` timestamps.

```php
>>> use GrahamCampbell\GitHub\Facades\GitHub;
>>> GitHub::api('apps')->findInstallations();
Github\Exception\RuntimeException with message ''Issued at' claim ('iat') must be an Integer representing the time that the assertion was issued'
```

There's no way to change this behavior in version `4.0.0` but an option will be added to version `4.1.0` (see [this pull request](https://github.com/lcobucci/jwt/pull/617)). Currently, a workaround in version `4.0.0` is to create timestamps with:
```php
// Reference: https://github.com/lcobucci/jwt/discussions/623#discussioncomment-262892_
$now = new DateTimeImmutable('@' . time())
```

When version `4.1.0` is finally released, you can also use the new `UnixTimestampDates` `ClaimsFormatter` using:
```php
$builder = $configuration->builder(ChainedFormatter::withUnixTimestampDates())
```

PS: I spent an hour tracking down this weird problem...

**Comment by GrahamCampbell** [comment]
Thanks for getting in touch. What version of this package are you using?

**Comment by GrahamCampbell** [comment]
This should be fixed in v9.8.2 and v10.0.2. You don't need to use `lcobucci/jwt` 4.1.x. The latest 3.4.x and 4.0.x should both work now.
